### ISSUE TITLE
QUESTION: Is is possible to load a Jpeg image in a byte array using LoadImage

### ISSUE THREAD

**Issue by juanp3053** [takuya-takeuchi/DlibDotNet#99]
Hello I just have a question,
Is it possible to load a Jpeg image in a byte array? Meaning that I would have a byte array representing a jpg/jpeg image and I would like to load it in the same fashion as 
`Array2D<T> LoadImage<T>(string path)` 
 but instead of passing the path as a parameter I would pass the byte array that represents the image.
Such as: 
`var img = Dlib.LoadImage<byte>(image);` 

Thanks in advance for the help, It is really appreciate it

**Comment by takuya-takeuchi** [comment]
I guess you want to get byte arrya as DECODED data, right?
No. TBH, I have never think I try to get color pixel data as byte array.
Perhaps, dlib does not provide any function you hope.

In this case, we can do

````
RgbPixel[] array = LoadImageAsMatrix<RgbPixel>(string path).ToArray();
````

Then, this array can be converted to byte array manually.

**Comment by juanp3053** [comment]
Thank you for you fast response!
I think I'm asking the wrong question here... let me try again.

I already have an image decoded to a byte array but now I want to 'Load it' into an Array2D so that I can use the FrontalFaceDetector. Is there a method to do that? I have tried 'loading it' with
`Array2D<T> LoadImageData<T>(byte[] data, uint rows, uint columns, uint steps)`
but it doesn't seem to convert well (when I Set the image on the ImageWindow it does not show correctly) or I get some random `"Attempted to read or write protected memory. This is often an indication that other memory is corrupt." Exception`

Maybe I'm just doing something very wrong, Thanks a lot for your help.

**Comment by takuya-takeuchi** [comment]
Ok, I understand.
What you did could be right.
If you saw broken or strange image on ImageWindow, data may be converted from Windows bitmap.

Could you check the this issue?
https://github.com/takuya-takeuchi/DlibDotNet/issues/48

**Comment by juanp3053** [comment]
That was my problem, I resolved it converting the byte array into a Windows bitmap that way using the `Array2D<T> LoadImageData<T>` method.
Thank you so much for the fast response.
